src/main/java/use_cases/add_edit_expenses_use_case/ExpenseUCI.java

```diff
@@ -227,33 +225,14 @@ public Category findCategory (ArrayList < Category > monthCategoryData, String n
             throw new NoSuchElementException();
         }
 
-    /**
-     * Identifying the only two cases when Session.recurringExpenseData needs to updated when attempting to edit Expense.
-     * (The user changes their response for the isRecurringExpense checkbox)
-     * @return boolean suggesting there is a need to update Session.recurringExpenseData
-     */
-    private boolean changeInRecurringInfo() {
```

Comment:
Please keep this I think this method is really great and helps keep code simpler in the main method!!